package.json

```diff
@@ -71,14 +71,16 @@
     "yam": "0.0.18"
   },
   "devDependencies": {
-    "babel-eslint": "^5.0.0",
+    "babel-eslint": "^6.0.4",
     "babel-plugin-transform-regenerator": "^6.5.2",
+    "babel-polyfill": "^6.9.0",
     "babel-preset-es2015": "^6.5.0",
     "babel-preset-stage-1": "^6.5.0",
     "babel-register": "^6.5.2",
     "chai": "^3.5.0",
     "chai-as-promised": "^5.2.0",
-    "eslint": "^2.2.0",
+    "eslint": "^2.10.0",
+    "estraverse-fb": "^1.3.1",
```

Comment:
Yeah, I don't think it is needed anymore.  Feel free to take it out and verify linting still works.